app/assets/stylesheets/header.sass

```diff
@@ -1,11 +1,49 @@
 #top
-  background: $black
+  background: $blue
```

Comment:
Name this something like `$primary` color? This color would change from blue to any other color the marketplace owner wants it to be.